### ISSUE TITLE
adkats not installing - missing tables

### ISSUE THREAD

**Issue by SpankDaddy** [AdKats/AdKats#943]
I've been over this again and again and I can't get Adkats running. I've installed the xpkiller tool and it's running without any issues. I've run the sql script on the same database, but when that's attempted, I get the following errors:

-----------------------------
REPLACE INTO `adkats_commands` VALUES(74, 'Active', 'player_pm_send', 'Unable', 'Player Private Message', 'msg', FALSE, 'Any');

REPLACE INTO `adkats_commands` VALUES(75, 'Active', 'player_pm_reply', 'Unable', 'Player Private Reply', 'r', FALSE, 'Any'); 

MySQL said: Documentation
#1064 - You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'REPLACE INTO `adkats_commands` VALUES(75, 'Active', 'player_pm_reply', 'Unable',' at line 3 
-----------------------------

Current PHPMyAdmin - Version information: 4.3.8

If I add them manually there are no issues getting them added. but when I run AdKats I see the following error:

-----------------------------
 [AdKats] ERROR-6900: [Non-Query failed. [Adding AdKats role groups table]: MySql.Data.MySqlClient.MySqlException: Cannot add foreign key constraint
   at PRoConEvents.AdKats.SafeExecuteNonQuery(MySqlCommand command)

[AdKats] ERROR-6900: [AdKats tables not present or valid in the database. Have you run the AdKats database setup script yet? If so, are your tables InnoDB?]
-----------------------------

It doesn't specifically state which tables are missing and the script will not install through the query. 

Any ideas? Anyone?

Thanks!

**Comment by SpankDaddy** [comment]
I've managed to fix this part:

 [AdKats] ERROR-6900: [Non-Query failed. [Adding AdKats role groups table]: MySql.Data.MySqlClient.MySqlException: Cannot add foreign key constraint
   at PRoConEvents.AdKats.SafeExecuteNonQuery(MySqlCommand command)

Now I'm still seeing the tables missing.

**Comment by ColColonCleaner** [comment]
Have you followed the install instructions to the letter?

https://github.com/AdKats/AdKats#install

The first error you were seeing is due to a bug in the latest version of PHPMyAdmin, we don't know why, but that valid syntax is not accepted. Run any other tool, such as mysql workbench, or a previous version of phpmyadmin, and that script will run.

It sounds like you tried to force it to work in the wrong order, please follow the install instructions. You will probably have to drop all the existing tables from your AdKats db to make sure nothing was corrupted by the previous install attempt.

**Comment by SpankDaddy** [comment]
Thanks, but I just managed to fix it. I looked into the sql file and noted that there was indeed a table not being added due to the script failing. I manually added the tables and re-started the layer and ran adkats again. It's now working. The tables that were not added were anything related to "adkats_records_main" and "adkats_records_debug" - Once those were in place, bam ADkats did the rest and it's working now. 

Weird shit. 

Thanks for the input and I'm sorry if I wasted anyones time.
